### PR TITLE
login registration leaderboard empty classes

### PR DESCRIPTION
**Pull request by arifamansoor**
All the empty classes without any implementation. *The test for the user entity will not work as user needs to be implemented.

### CODE REVIEW

**Review by FFFiend — APPROVED** [review]
Looks good for the file structure of the login-leaderboard

**Review by tseamuscorlett — CHANGES_REQUESTED** [review]
As I commented in the Discord - no need to push the AssetSetter class.
You can commit & push without selecting that class, or just correct it as follows:

The change was MapHazards -> MazeHazards 
But you changed AssetSetter -> MazeHazards

**Review by tseamuscorlett — APPROVED** [review]

File: src/main/java/adapters/login_leaderboard/LeaderboardPresenter.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package adapters.login_leaderboard;
+
+public class LeaderboardPresenter {
+    // Transforms the information from all three leaderboards into a viewable format.
```
Comment (enerko) [inline src/main/java/adapters/login_leaderboard/LeaderboardPresenter.java:4]: Javadoc should be written before the class, in the format of /** **/

**Review by enerko — CHANGES_REQUESTED** [review]
You should be good to go if you just fix the javadoc!

File: src/main/java/adapters/login_leaderboard/LeaderboardPresenter.java (hunk 1 above)
Comment (enerko) [reply src/main/java/adapters/login_leaderboard/LeaderboardPresenter.java:4]: Also just a side note: commit messages shouldn't be repeated, as it becomes difficult for others to see what you changed in the commit. It doesn't have to be long, just make sure that the commit message describes what you changed from next time!

**Review by tseamuscorlett — APPROVED** [review]

**Pull request merged**